components/brave_wallet_ui/common/constants/urls.ts

```diff
@@ -3,4 +3,10 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // you can obtain one at http://mozilla.org/MPL/2.0/.
 
+import { loadTimeData } from '../../../common/loadTimeData'
+
 export const FILECOIN_FORMAT_DESCRIPTION_URL = 'https://lotus.filecoin.io/lotus/manage/lotus-cli/#lotus-wallet-import'
+
+export const IMAGE_BRIDGE_URL = loadTimeData
+  .getString('braveWalletImageBridgeUrl')
```

Comment:
please add a check here that this string ends in `/` and only slice if it's true

Reply:
check added